### DISCUSSION TITLE
Database under maintenance

### DISCUSSION THREAD

**Discussion by rondon-d** [lbbe-software/seqinr#18] [General]
Hello, 
I am trying to load some data base, but it is still under maintenance. Do you think dataset will comeback? or has it been replace by some other one.  

choosebank("ensfungi") # select a data set 
Error in acnucopen(bank, socket) : 
  Database with name -->ensfungi<-- is currently off for maintenance, please try again later.

Thank you for your time :)

**Comment by simonpenel** [comment]
Hello,
sorry for this problem.
Actually we planed to stop maintaining some of the databases because the cost-benefit of the maintenance is weak, we will have to update the list of available databases. Typically for the  Ensembl databases , since Ensembl propose some tools for handle the data, and since the  the maintenance of an "acnuc" version implies a lot of disk space and calculation, we planed to stop their maintenance.

If you think that the access to these databases via seqinr is essential for you, let me know and we will think about a solution.

all the best

Simon

**Comment by rondon-d** [comment]
Hello, 
I have found a solution for this, on the _embl_ database.
Thank you for the information :) 

Warmly!
--
Diego

**Comment by simonpenel** [comment]
Hello,
for your information the databases ensembl are  online again, unfortuntaly the last release is quiet old.  ( I suppose it may still be usefull for teaching purpose.)
